src/main/java/pl/javastart/task/model/Processor.java

```diff
@@ -0,0 +1,26 @@
+package pl.javastart.task.model;
+
+public class Processor extends OverclockableComponent {
+
+    public Processor(String producer, String model, String serialNumber, int frequency,
+                     double currentTemperature, double maxSafeTemperature) {
+        super(producer, model, serialNumber, frequency, currentTemperature,
+                maxSafeTemperature);
+        this.temperatureIncreasePer100Mhz = 10;
+    }
+
+    @Override
+    public void overClock(double additionalFrequency) {
+        super.overClock(additionalFrequency);
+    }
```

Comment:
Jeżeli w ramach metody wywołujesz tylko metodę z klasy nadrzędnej przez super, to nie ma sensu zapisywać takiej metody.
Tę metodę i metodę z klasy Memory możesz usunąć